### ISSUE TITLE
selectableRowsHideCheckboxes hides check boxes but still takes up space

### ISSUE THREAD

**Issue by aryo** [gregnb/mui-datatables#1591]
When `selectableRowsHideCheckboxes` is set to true, the first column containing the check boxes is still there (even though the checkboxes are no longer visible).

Is there a way to remove the whole column? or at least make it take up 0 space?

## Expected Behavior
Checkbox column does not take up space

## Current Behavior
Checkbox column takes up space even though checkboxes are hidden

## Steps to Reproduce (for bugs)
https://codesandbox.io/s/muidatatables-custom-toolbar-kwemy?file=/index.js

Play around with `selectableRowsHideCheckboxes` option

## Your Environment

| Tech         | Version |
|--------------|---------|
| Material-UI  | 4.11.0        |
| MUI-datatables  |   3.7.1      |
| React        |   16.13.1      |
| browser      |         |
| etc          |         |

**Comment by aryo** [comment]
I've had to override the theme based on whether the `selectableRowsHideCheckboxes` option is true to get this to work. Ideally this should happen out of the box if the option is true.

```
MUIDataTableSelectCell: {
      root: {
        display: "none",
      }
    }
``

**Comment by wdh2100** [comment]
https://github.com/gregnb/mui-datatables/blob/fde3497a552040505f70c274466715fce1d7e8ab/src/components/TableSelectCell.js#L165

I think it's in the wrong place.

**Comment by wdh2100** [comment]
@aryo 

Fixed 3.7.2

**Comment by aryo** [comment]
@wdh2100 Good stuff!
